recipes/jupyter-rfb/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "jupyter-rfb" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_rfb-{{ version }}.tar.gz
+  sha256: 4c2cbab6b5c81379f30ccad3906c875c4d59d4758a36bb75805fb920d5b0bcf8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - jupyter-packaging >=0.7.9,<0.8
+    - jupyterlab >=3.0,<4
```

Comment:
unless you really need it, it's often most robust to _not_ require `jupyterlab` at build time. Luckily `jupyter-packaging` is no longer a runtime dep of `lab`, but... things happen.

also, if you _only_ support `jupyterlab >=X`, but don't want to make it an explicit dependency (which is fine), you can specify:

```yaml
requirements:
  run_constrained:
    - jupyterlab >=X
```

Reply:
Thanks. I didn't know about run_constrained. I'll try that out. I think I only added the jupyterlab dependency after the builds weren't working the first time.